docs/guides/admin/scaling_seat.md

```diff
@@ -23,7 +25,7 @@ So, which components are there to SeAT?
 
 Each of these components can live on their own [server](#server) and must share the same [immutable resources](#immutable-resources).
 
-## immutable resources
+## Immutable resources
 While almost every component in SeAT can 'run on its own', there are some services that SeAT consumes that SeAT can not scale itself. Instead, SeAT can consume a clustered or load balanced instance of these services. There are various reasons for this where the most important is that state is maintained between queue workers using Redis and MySQL.
```

Comment:
There is MySQL mentioned, should be replaced by maraiadb, or make it general: Database